DetectorDescription/Detectors/src/SimpleParallelogram_geo.cxx

```diff
@@ -0,0 +1,65 @@
+// $Id: $
+//====================================================================
+//  AIDA Detector description implementation for FCC
+//-------------------------------------------------------------------- 
+// Simple parallelogram using dimensions to be used to define cone composed of 1 single material
+//
+// @author Clement Helsens
+//====================================================================
+#include "DD4hep/DetFactoryHelper.h"
+
+using namespace std;
+using namespace DD4hep;
+using namespace DD4hep::Geometry;
+
+
+static Ref_t create_element(LCDD& lcdd, xml_h e, SensitiveDetector sens)  {
```

Comment:
Same comments as for SimpleCone_geo.cxx: Namespace, name and unused parameter warning.